### ISSUE TITLE
Hitbox parameter for particle projectile [Enhancement]

### ISSUE THREAD

**Issue by TheKot1k** [magemonkeystudio/fabled#390]
**Problem:**
I created a skill that allows the player to shoot a projectile, but the main problem is that I cannot reduce the hitbox of the projectile and it ruins everything.

**Solution:**
Adding a hitbox parameter for particle projectile.

**Screenshots:**
I deal damage by shooting off target XD
![изображение_2022-09-25_014902436](https://user-images.githubusercontent.com/94331370/192121230-91c2313e-90a2-4744-a676-7114e6b24997.png)
![image](https://user-images.githubusercontent.com/94331370/192121236-f89f8ba0-dcd9-4a38-b9d1-ffa4b58a1039.png)

**Comment by Sentropic** [comment]
Is this a Particle Projectile? If so, there's the tolerance parameter which is intended for that

**Comment by lordsrubka5** [comment]
I remember tolerance being a thing a while ago though now I cant seem to locate it in the config or within skill files and search is giving no results

**Comment by Sentropic** [comment]
Yeah my bad, I thought Particle Projectiles had it, but it's just the Single Target 😅 this is definitely a necessity

**Comment by Travja** [comment]
Sentropic completed this quite a while ago.
